util/ubuntu-setup

```diff
@@ -156,6 +160,11 @@ fi
 # install all of the packages listed in required_packages.txt
 apt-get install $ARGS $(sed 's/#.*//;/^$/d' $BASE/util/ubuntu-packages.txt)
 
+if [ "$SYSTEM" = "ubuntu-14.04" ]; then
```

Comment:
It dosent actually matter, both work. `=` is legacy, so its more 'cross platform', but I prefer to use `==` because I don't want to mess up in other languages. Yah...